TemplateManagement/PowerShell/ActiveDirectoryJoinV2/Join-Domain-StudentVM.ps1

```diff
@@ -138,7 +138,7 @@ try {
 
      Write-LogFile "Clean up section."
      Remove-Item -Path $passwordPath -Force
-     Remove-Item -Path "C:\Users\Public\Documents\DomainJoin.ps1" -Force
+     Remove-Item -Path "$env:PUBLIC\Documents\DomainJoin.ps1" -Force
```

Comment:
Where is this script, "DomainJoin.ps1"?  I searched the scripts but don't see that we're writing this?